Content.Server/Station/Components/StationFrequencyComponent.cs

```diff
@@ -0,0 +1,11 @@
+namespace Content.Server.Station.Components;
+
+/// <summary>
+/// Allows to set the frequency of all outgoing messages from the grid for specific channels.
+/// </summary>
+[RegisterComponent]
+public sealed partial class StationFrequencyComponent : Component
+{
+    [DataField]
+    public Dictionary<string, int> Frequency = new();
+}
```

Comment:
This isn't documented and it's not really clear what it's even doing, also is there a good reason why channel frequencies are being tied to the station and not the channel prototype.

Reply:
Right now there is no tool how to make the grids on the same map not hear each other (which could be useful), so I tried to implement it this way.